code/control/DeployDispatcher.php

```diff
@@ -172,25 +172,31 @@ public function save(\SS_HTTPRequest $request) {
 	 * @return \SS_HTTPResponse
 	 */
 	public function start(SS_HTTPRequest $request) {
+
```

Comment:
personally I don't like the additions of newlines in the first line of methods... :)